domain/robstoll-lib/extensions/kotlin_1_3/atrium-domain-robstoll-lib-kotlin_1_3-common/src/main/kotlin/ch/tutteli/atrium/domain.robstoll.lib.kotlin_1_3/creating/resultAssertions.kt

```diff
@@ -1,5 +1,15 @@
 package ch.tutteli.atrium.domain.robstoll.lib.kotlin_1_3.creating
 
-fun _isSuccess() {
-    TODO("fix signature and implement, only here so that module.info does not complain")
-}
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.ExpectImpl
+import ch.tutteli.atrium.domain.creating.changers.ExtractedFeaturePostStep
+import ch.tutteli.atrium.translations.DescriptionResultAssertion
+
+fun  <E, T : Result<E>>_isSuccess( assertionContainer: Expect<T>) : ExtractedFeaturePostStep<T, E> =
+    ExpectImpl.feature.extractor(assertionContainer)
+        .withDescription(DescriptionResultAssertion.VALUE)
+        .withRepresentationForFailure(DescriptionResultAssertion.ILLEGAL_STATE_EXCEPTION)
```

Comment:
please use IS_NOT_SUCCES (as it was before) is more descriptive than ILLEGAL_STATE_EXCEPTION